index.js

```diff
@@ -18,20 +29,199 @@ async function loadCountriesData() {
     }, {});
 }
 
+const baseUrl = 'https://restcountries.com/v3.1/alpha?fields=borders&fields=cca3&codes=';
 const form = document.getElementById('form');
 const fromCountry = document.getElementById('fromCountry');
 const toCountry = document.getElementById('toCountry');
 const countriesList = document.getElementById('countriesList');
 const submit = document.getElementById('submit');
 const output = document.getElementById('output');
+let countriesData;
+const codeByCountryDict = [];
+let watchedCountries = new Set();
+let graph = [];
+let currentRoute = [];
+let layerCounter = 0;
 
-(async () => {
-    fromCountry.disabled = true;
-    toCountry.disabled = true;
-    submit.disabled = true;
+// По уже скачанному массиву данных создаёт строки маршрута методом поиска в глубину.
+function CreateRoute(routes, to) {
```

Comment:
С большой буквы пишутся классы, синглтоны, модули и всё такое.
```suggestion
function createRoute(routes, to) {
```

Reply:
Да. Проглядел. C# не отпускает.